2696. Minimum String Length After Removing Substrings/solution.py

```diff
@@ -9,3 +9,5 @@
 		# in both loops, if a match is found, cut those two characters out of the string and reform as a new string
 			# if this happens decrement strlen by 2
```

Comment:
Oh this approach is much different from ours and I think it's quite innovative! However it becomes a bit of a pain to keep track of what has already been subtracted from the string, so I think you should remove the substring from the string once you find it and then return the length of the string without necessarily needing to keep track of it.